insights/specs/datasources/db2.py

```diff
@@ -0,0 +1,60 @@
+"""
+Custom datasources to get the db2 users and database names.
+"""
+
+from insights.core.context import HostContext
+from insights.core.exceptions import SkipComponent
+from insights.core.plugins import datasource
+from insights.combiners.ps import Ps
+from insights.core.spec_factory import foreach_execute
+from insights.specs import Specs
+
+
+@datasource(Ps, HostContext)
+def db2_users(broker):
+    """
+    This datasource get the name of the users running db2 databases
+
+    Returns:
+        list: the names of the users running db2 databases
+
+    Raises:
+        SkipComponent: there is no db2 users running db2 databases
+    """
+    ps = broker[Ps]
+    ps_list = ps.search(COMMAND_NAME__contains="db2sysc")
+    if ps_list:
+        users_result = set(filter(None, [item.get("USER") for item in ps_list]))
+        if users_result:
+            if "root" in users_result:
+                users_result.remove("root")
```

Comment:
Just in case there is only `root` in the results, how about changing the above lines as 
```python
users_result.discard('root')
if users_result:
    return sorted(users_result)
```

Reply:
@xiangce - Updated, please check